roundabout/piece.js

```diff
@@ -57,13 +56,13 @@ export function asCarCid(cid) {
 }
 
 /**
- * Find the set of CAR CIDs that are claimed to be equivalent to the Piece CID.
+ * Find the set of CIDs that are claimed to be equivalent to the Piece CID.
  * 
  * @param {CID} piece
  * @param {(CID) => Promise<Claim[]>} [fetchClaims] - returns content claims for a cid
  */
-export async function findEquivalentCarCids (piece, fetchClaims = createClaimsClientForEnv()) {
-  /** @type {Set<CARLink>} */
+export async function findEquivalentCids (piece, fetchClaims = createClaimsClientForEnv()) {
+  /** @type {Set<import('multiformats').UnknownLink>} */
   const cids = new Set()
```

Comment:
Hmm, I think this is not going to dedupe like you think it will.

Reply:
yeah, I also think this set will probably not guarantee . will create an issue, as this was like it and look after